spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/KafkaStreamsStreamListenerSetupMethodOrchestrator.java

```diff
@@ -431,4 +479,24 @@ private boolean isDeclarativeInput(String targetBeanName, MethodParameter method
 		}
 		return null;
 	}
+
+	@SuppressWarnings({"unchecked"})
+	private static KafkaStreamsStateStoreProperties getStateStoreSpec(Method method) {
```

Comment:
The name of the method doesn't seem to match functionality it provides since you are not 'getting', rather 'building' and/or 'extracting', so perhaps `buildStateStoreSpec(..)`.